### DISCUSSION TITLE
Types issue with vite config possibly caused by multiple vite versions

### DISCUSSION THREAD

**Discussion by sgcullen** [wxt-dev/wxt#1273] [General]
Hi Aaron and team,
I am getting a types issue but I believe it's being casued by multiple versions of Vite being installed (6.0) and also 5.4.x (vitest).
<img width="979" alt="image" src="https://github.com/user-attachments/assets/1a276fd1-7526-4cd6-8efc-4f409a424829" />

Does anyone have a preferred/ recommended way to handle this at the moment? I'm using pnpm:

<img width="364" alt="image" src="https://github.com/user-attachments/assets/89a1a164-ee24-485e-a855-687cec43fea8" />

Thanks as always!

**Comment by aklinker1** [comment]
I'd recommend installing vite 5 until vitest adds support. Just running `npm i vite@^5.4.11` should do the trick

**Reply by sgcullen** [reply]
Thanks Aaron, yes you're right.
